src/components/input/input.js

```diff
@@ -480,6 +530,7 @@ function placeholderDirective($log) {
   }
 }
 
+var visibilityDirectives = ['ngIf', 'ngShow', 'ngHide', 'ngSwitchWhen', 'ngSwitchDefault'];
```

Comment:
I actually thought about this, and don't think it makes sense to use these responsive directives here.